### PR TITLE
Log formatted files

### PR DESCRIPTION
**Pull request by challen0**
This address #58 and prints a log message when a file is formatted. I put the message on the info level since the other messages about solutions/projects being formatted were that level too.

### CODE REVIEW

File: src/Resources.resx [hunk 1]
```diff
@@ -189,4 +189,7 @@
   <data name="The_files_to_operate_on_If_none_specified_all_files_in_workspace_will_be_operated_on" xml:space="preserve">
     <value>The files to operate on. If none specified, all files in workspace will be operated on.</value>
   </data>
+  <data name="Formatted_code_file_0" xml:space="preserve">
+    <value>Formatted '{0}'.</value>
```
Comment (JoeRobich) [inline src/Resources.resx]: This should be changed so that the name matches the value.

File: tests/CodeFormatterTests.cs [hunk 2]
```diff
@@ -156,5 +156,22 @@ public async Task NoFilesFormattedWhenNotInList()
...
+        public async Task OnlyLogFormattedFiles()
+        {
+            var logger = new TestLogger();
+            var path = Path.GetFullPath("tests/projects/for_code_formatter/unformatted_solution/unformatted_solution.sln", SolutionPath);
+
+            var files = new[] {"Program.cs"};
+            var formatResult = await CodeFormatter.FormatWorkspaceAsync(logger, path, isSolution: true, logAllWorkspaceWarnings: false, saveFormattedFiles: false, filesToFormat: files, cancellationToken: CancellationToken.None);
+            var log = logger.GetLog();
+
+            var pattern = string.Format(Resources.Formatted_code_file_0, @"Program\.cs");
```
Comment (JoeRobich) [inline tests/CodeFormatterTests.cs]: I you formatted with the string `(.*)` instead, you could match with the pattern to extract the formatted file names out of the log.

File: src/Resources.resx (hunk 1 above)
Comment (challen0) [reply src/Resources.resx]: Thanks for pointing that out! I missed some text when I changed it.

File: tests/CodeFormatterTests.cs (hunk 2 above)
Comment (challen0) [reply tests/CodeFormatterTests.cs]: I think I understand what you're saying, but I made a slight adjustment to the test to find the file name.

**Pull request merged**